alpine.dockerfile

```diff
@@ -25,49 +25,29 @@ LABEL maintainer="Microsoft" \
       org.label-schema.vcs-url="https://github.com/Azure/azure-cli.git" \
       org.label-schema.docker.cmd="docker run -v \${HOME}/.azure:/root/.azure -it mcr.microsoft.com/azure-cli:$CLI_VERSION"
 
-# bash gcc make openssl-dev libffi-dev musl-dev - dependencies required for CLI
-# openssh - included for ssh-keygen
-# ca-certificates
 
-# curl - required for installing jp
-# jq - we include jq as a useful tool
-# pip wheel - required for CLI packaging
-# jmespath-terminal - we include jpterm as a useful tool
-# libintl and icu-libs - required by azure devops artifact (az extension add --name azure-devops)
-
-# We don't use openssl (3.0) for now. We only install it so that users can use it.
-RUN apk add --no-cache bash openssh ca-certificates jq curl openssl perl git zip \
- && apk add --no-cache --virtual .build-deps gcc make openssl-dev libffi-dev musl-dev linux-headers \
- && apk add --no-cache libintl icu-libs libc6-compat \
- && apk add --no-cache bash-completion \
- && update-ca-certificates
+# ca-certificates bash bash-completion jq jp openssh-keygen - for convenience
+# libintl icu-libs - required by azure-devops https://github.com/Azure/azure-cli/pull/9683
+# libc6-compat - required by az storage blob sync https://github.com/Azure/azure-cli/issues/10381
+# gcc python3-dev musl-dev linux-headers libffi-dev - temporarily required by psutil
```

Comment:
`python3-dev` is not in the original `alpine.dockerfile`. Is it really needed?

Reply:
When install `psutil` in alpine, it prints
```
psutil could not be installed from sources because gcc is not installed. Try running:
sudo apk add gcc python3-dev
```
Weird, it can be built without `python3-dev`.